simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogPagedViewHolder.kt

```diff
@@ -508,16 +508,6 @@ class CatalogPagedViewHolder(
           heightMatchParent = true
         )
       )
-    } else if (isBookDeletable(book)) {
-      this.idleButtons.addView(this.buttonCreator.createButtonSpace())
-      this.idleButtons.addView(
-        this.buttonCreator.createDeleteButton(
-          onClick = {
-            this.listener.delete(this.feedEntry as FeedEntryOPDS)
-          },
-          heightMatchParent = true
-        )
-      )
```

Comment:
Can you explain why this was deleted, Instead of keeping it, but labeling it "Return" like you did in other places? I'm not saying it's wrong, I'm just not sure about it.

Reply:
I've missed it! It was supposed to be restored as well. Thanks for pointing that out.